arch/riscv/Makefile.in

```diff
@@ -0,0 +1,62 @@
+# Makefile for zlib-ng
+# Copyright (C) 1995-2013 Jean-loup Gailly, Mark Adler
+# Copyright (C) 2024 Hans Kristian Rosbach
```

Comment:
File added in 2025 should have copyright line with year or year range ending in 2025 listing person or organisation who did last major edit.